codeship-steps.yml

```diff
@@ -3,7 +3,7 @@
   steps:
     - name: Test Deployment
       service: deployment
-      command: /deploy/test/deploy_to_google.sh
+      command: /test/deploy_to_google.sh
```

Comment:
The `deployment` folder inside the repository is mounted as a volume to `/deploy` (inside the container) via https://github.com/codeship-library/google-cloud-utilities/blob/master/codeship-services.yml#L9